web3.js/src/connection.ts

```diff
@@ -36,6 +35,7 @@ import fetchImpl, {Response} from './fetch-impl';
 import {DurableNonce, NonceAccount} from './nonce-account';
 import {PublicKey} from './publickey';
 import {Signer} from './keypair';
+import RpcWebSocketClient from './rpc-websocket';
```

Comment:
We create `RpcWebSocketClient` like this later:
```
this._rpcWebSocket = new RpcWebSocketClient(this._rpcWsEndpoint, {
      autoconnect: false,
      max_reconnects: Infinity,
    });
```
it seems that `max_reconnects: Infinity` is a bit misleading here, as it is going to be overwritten by explicitly hardcoded `max_reconnects: 5` in the `rpc-websocket.ts` file?